src/encoded/static/components/matrix_mouse_development.js

```diff
@@ -155,6 +155,43 @@ function sortMouseArray(a, b) {
     return aNumerical - bNumerical;
 }
 
+// Generate object listing all ages corresponding to a mouse stage based on rowCategoryData
+const getMouseAgeObject = (rowCategoryData) => {
```

Comment:
"Hungarian notation" notation is frowned upon in the JavaScript community (or at least those that care about Computer History so are aware of it)

Reply:
Yes, it makes sense not to include "object" in the name because it should be evident that it is an object.